### PR TITLE
test: reduce line length to 140

### PR DESCRIPTION
**Pull request by jelly**
This is a middle step of moving the tons of long lines to a more reasonable default.

### CODE REVIEW

File: test/check-machines-create [hunk 1]
```diff
@@ -1633,8 +1641,10 @@ vnc_password= "{vnc_passwd}"
             root.find('os').find('resources').find('minimum').find('storage').text = '134217728'
             root.find('os').find('eol-date').text = '9999-12-31'
             new_fedora_28_xml = ET.tostring(root)
-            self.machine.execute(f"echo '{str(new_fedora_28_xml, 'utf-8')}' > {self.test_obj.vm_tmpdir}/fedora-28.xml")
-            self.machine.execute(f"mount -o bind  {self.test_obj.vm_tmpdir}/fedora-28.xml /usr/share/osinfo/os/fedoraproject.org/fedora-28.xml")
+            self.machine.execute(f"""
+            echo '{str(new_fedora_28_xml, 'utf-8')}' > {self.test_obj.vm_tmpdir}/fedora-28.xml
```
Comment (martinpitt) [inline test/check-machines-create:1645]: In your next step: Split this up and use self.machine.write()

File: test/check-machines-networks [hunk 2]
```diff
@@ -219,7 +229,7 @@ class TestMachinesNetworks(machineslib.VirtualMachinesCase):
                         for xfo in self.xfail_objects:
                             if "range" in xfo:
                                 b.wait_in_text(
-                                    f"#create-network-dialog .create-network-{xfo} .pf-v5-c-form__helper-text .pf-v5-c-helper-text__item.pf-m-error",
+                        f"#create-network-dialog .create-network-{xfo} .pf-v5-c-form__helper-text .pf-v5-c-helper-text__item.pf-m-error",
```
Comment (martinpitt) [inline test/check-machines-networks:232]: This is too outdented now, and IMHO worse than before --  better split up the string to two lines

File: test/check-machines-virtualization [hunk 3]
```diff
@@ -60,9 +60,10 @@ class TestMachinesVirtualization(machineslib.VirtualMachinesCase):
 
         def stopLibvirtSocket():
             m.execute(
-    f"systemctl stop {self.getLibvirtServiceName()}-ro.socket {self.getLibvirtServiceName()}.socket {self.getLibvirtServiceName()}-admin.socket")
+f"systemctl stop {self.getLibvirtServiceName()}-ro.socket {self.getLibvirtServiceName()}.socket {self.getLibvirtServiceName()}-admin.socket")  # noqa: E501
```
Comment (martinpitt) [inline test/check-machines-virtualization:63]: This should also be fixed in the next round.

**Review by martinpitt — APPROVED** [review]
Thanks!

File: test/check-machines-create (hunk 1 above)
Comment (jelly) [reply test/check-machines-create:1645]:
```
test/check-machines-consoles:        self.machine.execute(f"echo '' > {args['logfile']}")
test/check-machines-create:            self.machine.execute("echo >>/etc/hosts 127.0.0.42 mirror.i3d.net")
test/check-machines-create:            self.machine.execute("echo >>/etc/hosts ::1 mirror.i3d.net")
test/check-machines-create:            self.machine.execute(f"echo '{str(new_rhel_8_1_xml, 'utf-8')}' > {self.test_obj.vm_tmpdir}/rhel-8.1.xml")
test/check-machines-create:            self.machine.execute(f"echo '{str(new_rhel_8_2_xml, 'utf-8')}' > {self.test_obj.vm_tmpdir}/rhel-8.2.xml")
test/check-machines-create:            self.machine.execute(f"echo '{str(new_rhel_7_1_xml, 'utf-8')}' > {self.test_obj.vm_tmpdir}/rhel-7.1.xml")
test/check-machines-lifecycle:        self.machine.execute(f"echo '{user_name}:foobar' | chpasswd")
test/check-machines-lifecycle:        self.machine.execute(f"echo '' > {args['logfile']}")
test/check-machines-lifecycle:        self.machine.execute(f"echo '' > {args2['logfile']}")
```

Note, we have more places we can use `self.machine.write`

Comment (jelly) [reply test/check-machines-create:1645]: I assume you mean `self.write_file`.

Comment (martinpitt) [reply test/check-machines-create:1645]: No, that will try to restore the file, but it's already in the vm_tmpdir.

Comment (jelly) [reply test/check-machines-create:1645]: Didn't read into the context, but it seems that we can generally cleanup the `execute('echo ...')` pattern.

File: test/check-machines-virtualization (hunk 3 above)
Comment (jelly) [reply test/check-machines-virtualization]: I've fixed this and other comments in the current PR as there is no time limit or rush to get this fixed.

**Review by martinpitt — APPROVED** [review]
Cheers!

**Pull request merged**